cypress/utils/utils.ts

```diff
@@ -835,6 +838,35 @@ export function createMultipleApplications(numberofapplications: number): Array<
     return applicationList;
 }
 
+export function createMultipleApplicationsWithBSandTags(
+    numberofapplications: number,
+    businessservice?: Array<BusinessServices>,
+    tagList?: Array<Tag>
+): Array<Application> {
+    var applicationList: Array<Application> = [];
+    var tags: string[];
+    var business: string;
+    clickByText(navMenu, applicationInventory);
+    for (let i = 0; i < numberofapplications; i++) {
+        if (!businessservice)
+            businessservice = createMultipleBusinessServices(numberofapplications);
+        business = businessservice[i].name;
+        if (tagList) tags = [tagList[i].name];
+        let appdata = {
+            name: data.getAppName(),
+            business: businessservice[0].name,
```

Comment:
Since it is inside a loop , It will take i index for BS and tags.
business: businessservice[i].name,